README.md

```diff
@@ -8,15 +8,15 @@ reSolve is a full stack functional JavaScript framework.
 
 - [CQRS](https://martinfowler.com/bliki/CQRS.html) - independent Command and Query sides.
 - [DDD Aggregate](https://martinfowler.com/bliki/DDD_Aggregate.html) support.
-- [Event sourcing](https://martinfowler.com/eaaDev/EventSourcing.html) - using events as a source of truth and calculates read models from them.
+- [Event sourcing](https://martinfowler.com/eaaDev/EventSourcing.html) - uses events as a source of truth and calculates read models from them.
```

Comment:
It is event sourc**ing**, so let's write "using events as a source of truth and calcula**ting** read models from them.

Reply:
ok, looks good